notebook.html

```diff
@@ -9,236 +9,271 @@
-			<p class="notebook-body-text"><strong>P & P:</strong>  On February 27th, P&P worked with YOURS to teach kids about physics and engineering principles by building paper towers to hold up as many books as possible. P&P regrouped to discuss this program and to continue brainstorming for future outreach events. P&P continued brainstorming other outreach events and ideas with the intention of reaching out to the kids at the local Ithaca High school.</p>
+<body class="notebook-page-wrapper">
+	<!------------------------ STANDARD PAGE BANNER START ------------------------>
+	<header class="standard-page-banner">
+		<div class="standard-page-banner-wrapper">
+			standardpagebanner
+			<div class="standard-page-banner-content-wrapper">
+				<p class="standard-page-banner-caption"></p>
+			</div>
 		</div>
-	</div>
-	<div class="notebook-content-wrapper">
-		<div class="notebook-header-wrapper"><p class="notebook-header-text">March 11th to March 17th<p></div>
-		<div class="notebook-text-wrapper">
-			<p class="notebook-body-text"><strong>Wet Lab:</strong>  Members split into training groups and performed PCR on gBlocks from last year’s project. Training involved overviews and run-throughs of cloning, transformation, miniprep, gel extraction, and other protocols.</p>
-			<p class="notebook-body-text"><strong>Wiki:</strong>  Wiki subteam learned the basics of HTML and CSS during a one-hour long Wiki/Design Training class. Worked on the HTML/CSS project (due 3/26).</p>
-			<p class="notebook-body-text"><strong>P & P:</strong>  P&P started planning and finding information for a synthetic biology discussion. Also brainstormed for activities for the annual student outreach event - Splash! at Cornell.</p>
-		</div>
-	</div>	<div class="notebook-content-wrapper">
-	<div class="notebook-header-wrapper"><p class="notebook-header-text">March 18th to March 24th<p></div>
-		<div class="notebook-text-wrapper">
-			<p class="notebook-body-text"><strong>Wiki:</strong>  Wiki subteam learned the basics of colors, white space, and composition during a one-hour long Wiki/Design Training class. Worked on webpage creation project to practice building information hierarchy (due 4/8). </p>
-			<p class="notebook-body-text"><strong>P & P:</strong>  P&P brainstormed applications for two possible projects for the team - incorporating synthetic biology with 3D printing or with fourier transforms/band-pass filters.</p>
+	</header>
+	<!------------------------ STANDARD PAGE BANNER END ------------------------>
+
+	<!------------------------ NOTEBOOK PAGE CONTENT START ------------------------>
+	<div class="notebook-page-content-wrapper">
+		<div class="notebook-content-wrapper">
+			<div class="notebook-header-wrapper"><p class="notebook-header-text">March 4th to March 10th<p></div>
+			<hr class="yellow-accent-line-center">
```

Comment:
After each `<div class="notebook-header-wrapper"><p class="notebook-header-text">March 4th to March 10th<p></div>`, can you add `<hr class="yellow-accent-line-center">` like I did above?